app/src/main/res/values/dimens.xml

```diff
@@ -625,4 +625,9 @@
   <dimen name="topic_lessons_title_margin_start">32dp</dimen>
   <dimen name="topic_lessons_title_margin_top">32dp</dimen>
   <dimen name="topic_lessons_title_margin_end">32dp</dimen>
+
+  <!-- Hints and Solution Fragment -->
+  <dimen name="hints_and_solution_recycler_view_padding_top">20dp</dimen>
+  <dimen name="hints_and_solution_recycler_view_padding_bottom">168dp</dimen>
+  <dimen name="hints_and_solution_toolbar_height">56dp</dimen>
```

Comment:
Suggest removing this and directly using `android:layout_height="?attr/actionBarSize"` in layout file.

Reply:
Okay. Actually I was using that only. Made the change to comply with the previous review.

I'll revert this.